backend/database_test.go

```diff
@@ -0,0 +1,266 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"github.com/go-redis/redismock/v8"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestConnect(t *testing.T) {
+	t.Run("should check that connection is already established and return error", func(t *testing.T) {
```

Comment:
You could name it this way:
`should return error when connection is already established`